Model/Source/DefaultOptions.php

```diff
@@ -181,21 +181,23 @@ public function getDigitalStampDefaultWeight()
     }
 
     /**
-     * Get package type
+     * Get package type ID by default, return package type name if $returnName is true
      *
-     * @return int 1|2|3|4
+     * @param  bool $returnName
+     *
+     * @return int|string
      */
-    public function getPackageType()
+    public function getPackageType(bool $returnName = false)
```

Comment:
Ik zou dit alsjeblieft in een aparte method doen: getPackageTypeName, die evt. deze weer aanroept, zodat het direct duidelijk is bij gebruik: getPackageTypeName returnt een string, getPackageType returnt altijd een int, en dat is dus de id dan, er is al veel verwarring mbt packageType door de hele plugin vind ik.